src/mapper/value-mapper.factory.ts

```diff
@@ -1,6 +1,7 @@
 import { config, email, filename, id, url, proxify, hashed, contentType } from '../anonymizer'
 import { TYPES } from './constants'
 import { string, boolean, number } from './transformers'
+import { extractDomains } from '../anonymizer/transformers/extract-domains.transformer'
```

Comment:
import it from `../anonymizer` module. In the same way as hahed, contentType, etc